allzpark/view.py

```diff
@@ -226,16 +229,18 @@ def addColumn(widgets, *args, **kwargs):
 
         for name, widget in docks.items():
             has_menu = hasattr(widget, "on_context_menu")
-            BtnCls = (dock.PushButtonWithMenu
-                      if has_menu else QtWidgets.QPushButton)
+            BtnCls = (dock.PushButtonWithMenu if has_menu
+                      else QtWidgets.QPushButton)
+            btn_icon = (widget.icon if isinstance(widget.icon, QtGui.QIcon)
+                        else res.icon(widget.icon))
```

Comment:
This seems related to #83.
Currently, the `res.icon` only look for images inside `allzpark.resources`, so the custom plugin can only create their `QIcon` by itself, hence adding this extra if-statement.